beta-src/src/utils/map/writeNotifications.ts

```diff
@@ -0,0 +1,97 @@
+import { withTheme } from "@emotion/react";
+import { rgbToHex } from "@mui/material";
+import { current } from "@reduxjs/toolkit";
+import { GameState } from "../../state/interfaces/GameState";
+
+/* eslint-disable no-param-reassign */
+export default function writeNotifications(state): void {
+  const {
+    ordersMeta,
+    overview: { phase },
+  }: {
+    // definition
+    ordersMeta: GameState["ordersMeta"];
+    overview: { phase: GameState["overview"]["phase"] };
+  } = current(state);
+  if (phase === "Builds") {
```

Comment:
Let's separate this logic out into its own file, something like "writeBuildNotifications" that way as we add different notifications it doesn't over saturate this file.